_faq/security_implications/gsuite-key.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Why does Forseti store the G Suite private key in the Compute Engine instnce?
+order: 1
+---
+{::options auto_ids="false" /}
+
+The 
```

Comment:
This doesn't really answer the question per-say. The real reason is the API requires certain oauth methods and the oauth library we use requires the key to be local to the code.